server.js

```diff
@@ -193,6 +193,12 @@ app.use(
           'www.googletagmanager.com',
           'www.google-analytics.com',
         ],
+        connectSrc: [
```

Comment:
Does this block need to be added for this PR to work correctly?

Reply:
It was an error I'd noticed and made sense to bring it in to the GA changes. But in theory, no...

However, I weighed this up vs more PRs and more e2e tests, etc...